src/msg/async/rdma/RDMAStack.h

```diff
@@ -185,8 +185,8 @@ class RDMAConnectedSocketImpl : public ConnectedSocketImpl {
  protected:
   CephContext *cct;
   Infiniband::QueuePair *qp;
-  ib_cm_meta_t peer_cm_meta;
-  ib_cm_meta_t local_cm_meta;
+  uint32_t peer_qpn = 0xFFFFFFFF;
+  uint32_t local_qpn = 0xFFFFFFFF;
```

Comment:
@tchaikov 
According to page 108 on spec "InfiniBandTM Architecture Specification Volume 1, Release 1.3" spec link: https://cw.infinibandta.org/document/dl/7859
The max queue pair number(qpn) ranges from "0" to "2^24 -1". So I initialize the peer_qpn/local_qpn to be 0xFFFFFFFF. After the queue pair created, peer_qpn/local_qpn will be set to the right value.

Reply:
> The architecture supports up to 2^24 QPs per channel adapter;

@iotcg this explains why we have `uint32_t` for qpn. but i don't think it makes sense to initialize it with `0xFFFFFFFF`.

but i don't feel strong about this though.

Reply:
@tchaikov 
It won't result in bug if we initialize the local_qpn/peer_qpn to be 0.
Should I initialize the local_qpn/peer_qpn to be 0 or 2^24?
Please note: qpn(queue pair number) range is different from psn(packet sequence number) though psn is also limited to be less than 2^24.

Reply:
i'd prefer 0.

Reply:
Done. local_qpn & peer_qpn are initialized to be 0